src/table/components/TableWrapper.jsx

```diff
@@ -54,10 +56,40 @@ export default function TableWrapper(props) {
   const { size, rows, columns } = tableData;
   const { page, rowsPerPage, rowsPerPageOptions } = pageInfo;
   const [focusedCellCoord, setFocusedCellCoord] = useState([0, 0]);
+  const [paginationHeight, setPaginationHeight] = useState();
   const shouldRefocus = useRef(false);
   const tableContainerRef = useRef();
   const tableWrapperRef = useRef();
-  const classes = useStyles();
+  const tablePaginationSectionRef = useRef();
+
+  useEffect(() => {
+    return () => {};
+  }, []);
+
+  useEffect(() => {
```

Comment:
Why move the useEffect?

Reply:
because of `const classes = useStyles({ paginationHeight }); ` want to have paginationHeight as soon as possible

Reply:
Does it make a difference? since it is a useEffect, does it matter where you define it? the function will run depending on the render cycle right?

Reply:
right, i had a try and there is no difference.